docs/src/main/asciidoc/security-oidc-protect-web-application-tutorial.adoc

```diff
@@ -0,0 +1,264 @@
+[id="security-protect-web-authentication-tutorial"]
+= Protect a web application by using OIDC authorization code flow mechanism
+include::_attributes.adoc[]
+:categories: security
```

Comment:
Consider adding `web` also. Some of the other web-related security topics also use this category too.
```suggestion
:categories: security,web
```